pkg/handler/asset.go

```diff
@@ -217,6 +217,13 @@ func (u *UserSelectHandler) proxyAsset(asset model.PermAsset) {
 			utils.IgnoreErrWriteString(u.h.term, lang.T("ACL reject"))
 			utils.IgnoreErrWriteString(u.h.term, utils.CharNewLine)
 			return
+		case model.ACLFaceVerify:
+			// todo: 需要人脸验证 后续需要发站内信通知用户，并且等待用户人脸验证通过
+			logger.Errorf("Create connect token and auth info failed: %s %s", tokenInfo.Code, tokenInfo.Detail)
+			msg := lang.T("Face verification is not supported yet. Please use the WebTerminal to connect the asset.")
+			utils.IgnoreErrWriteString(u.h.term, msg)
+			utils.IgnoreErrWriteString(u.h.term, utils.CharNewLine)
+			return
 		case model.ACLReview:
 			reviewHandler := LoginReviewHandler{
 				readWriter: u.h.sess,
```

Comment:
None found.

Regarding knowledge cutoff, we're currently at February 28th of the year 2024 so there should be no such problems.